skglm/solvers/group_bcd.py

```diff
@@ -163,6 +190,32 @@ def _bcd_epoch(X, y, w, Xw, lipschitz, datafit, penalty, ws):
                 Xw += (w[j] - old_w_g[idx]) * X[:, j]
 
 
+@njit
+def _bcd_epoch_sparse(X_data, X_indptr, X_indices, y, w, Xw, lipschitz, datafit, penalty, ws):
+    # perform a single BCD epoch on groups in ws
+    grp_ptr, grp_indices = penalty.grp_ptr, penalty.grp_indices
+
+    for g in ws:
+        grp_g_indices = grp_indices[grp_ptr[g]: grp_ptr[g+1]]
+        old_w_g = w[grp_g_indices].copy()
+
+        lipschitz_g = lipschitz[g]
+        grad_g = datafit.gradient_g_sparse(X_data, X_indptr, X_indices, y, w, Xw, g)
+
+        # X = sparse.csc_matrix((X_data, X_indices, X_indptr)).todense()
+        # grad_g_ = datafit.gradient_g(X, y, w, Xw, g)
+
+        w[grp_g_indices] = penalty.prox_1group(
+            old_w_g - grad_g / lipschitz_g,
```

Comment:
if possible harmonize code style, we don't break line before the closing parenthesis (but we have no strict guidelines on this)